js/SearchResult.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
in JS classes always run in strict mode, so there is no use for adding 'use strict' here